pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8270-composite-add-tabular-separator.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="sampath" id="20220704-add-composite-tabular-separator-column">
+        <addColumn tableName="composite_question">
+            <column name="tabular_separator" type="varchar(30)"/>
+        </addColumn>
+    </changeSet>
```

Comment:
it looks like this migration has been added twice- an identical `addColumn` element already exists in [DDP-8270-composite-add-separator.xml](https://github.com/broadinstitute/ddp-study-server/pull/2000/files#diff-611858b51e292ed8b3dea56ecffbfb2ff6ad23a9554aa7ed3d500b5db97ab6da)

Reply:
old not referenced should have been deleted. thank you